src/main/java/org/mockbukkit/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -953,15 +868,8 @@ public void attack(@NotNull Entity target)
 	{
 		Preconditions.checkNotNull(target, "Target cannot be null");
 
-		if (this instanceof Player)
-		{
-			((Player) this).attack(target);
-		}
-		else
-		{
-			// TODO Auto-generated method stub
-			throw new UnimplementedOperationException();
-		}
+		// TODO Auto-generated method stub
+		throw new UnimplementedOperationException();
```

Comment:
Was gonna comment on that this removes behavior, but then I realized the previous behavior is stackoverflow, so 👍

Reply:
Yeah, I got this during a test.